pkg/transparentproxy/transparentproxy_experimental.go

```diff
@@ -140,7 +144,7 @@ func (tp *ExperimentalTransparentProxy) Setup(tpConfig *config.TransparentProxyC
 		}
 	}
 
-	ipv6, err := ShouldEnableIPv6()
+	ipv6, err := ShouldEnableIPv6(redirectInboundPortIPv6)
```

Comment:
Maybe we should fix it also here: https://github.com/kumahq/kuma/blob/master/pkg/transparentproxy/istio/tools/istio-iptables/pkg/cmd/root.go#L179 if it hasn't been deprecated yet